ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -7,12 +7,15 @@ set -x
 set -e
 
 . util/build_consts.sh
-SHA=$(git rev-parse HEAD)
-BITCACHE_DIR=~/.cache/opentitan-bitstreams/cache/${SHA}
+readonly SHA=$(git rev-parse HEAD)
+readonly BIT_CACHE_DIR="~/.cache/opentitan-bitstreams/cache/${SHA}"
+readonly BIT_SRC_PREFIX="${BIN_DIR}/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit"
+readonly BIT_DST_PREFIX="${BIT_CACHE_DIR}/lowrisc_systems_chip_earlgrey_cw310_0.1.bit"
+
 mkdir -p $BITCACHE_DIR
-BITCACHE_FILE=$BITCACHE_DIR/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.orig
-cp $BIN_DIR/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit ${BITCACHE_FILE}
-echo "" >> ${BITCACHE_DIR}/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.splice
+for suffix in .orig .splice; do
+  cp "${BIT_SRC_PREFIX}.${suffix}" "${BIT_DST_PREFIX}.${suffix}"
```

Comment:
I think this file has had the suffix rewritten by the splice job to maintain compatibility with the other jobs. You might have to change it there if you want to access it with the suffix as you do above.

Reply:
You're right, this is copied to a path with the suffix during the splice step